### PR TITLE
Remove unused imports from Idris.Main

### PR DESCRIPTION
**Pull request by fabianhjr**

### CODE REVIEW

File: src/Idris/Main.idr [hunk 1]
```diff
@@ -1,35 +1,27 @@
 module Main
 
-import Core.Binary
-import Core.Context
+import Data.Strings
+import System
```
Comment (keram) [inline src/Idris/Main.idr:4]: Is there a convention on ordering imports in Idris ? I feel like the version before this change followed non-strict alphabetical order so i just wonder if it is convention or personal taste ?

Comment (fabianhjr) [reply src/Idris/Main.idr:4]: I wait, didn't mean to commit this, will ammend.

Comment (keram) [reply src/Idris/Main.idr]: haha, didn't realised this was mistake. But good work in general 👍

**Comment by edwinb** [comment]
This one probably needs the CI to restart since the import fixing patch might have caught some problems...

**Comment by fabianhjr** [comment]
Rebased all my open PRs and pushed.

**Pull request merged**